src/sentry/integrations/discord/utils/auth.py

```diff
@@ -0,0 +1,12 @@
+from cryptography.exceptions import InvalidSignature
+from cryptography.hazmat.primitives.asymmetric.ed25519 import Ed25519PublicKey
+
+
+def verify_signature(public_key_string: str, signature: str, message: str) -> bool:
+    try:
+        # https://cryptography.io/en/latest/hazmat/primitives/asymmetric/ed25519/#cryptography.hazmat.primitives.asymmetric.ed25519.Ed25519PublicKey
```

Comment:
Is there something specific in the docs you'd like to reference here? If not, future users can simply look up the library docs. 

I'd add a sentence with the info that's important in this case.

Reply:
That is a good point. I suppose this link really doesn't add anything!